From 138f78f34ee11f40daedf321067dc49bb9e9898a Mon Sep 17 00:00:00 2001
From: Alan Tull <r80115@freescale.com>
Date: Thu, 8 Apr 2010 11:13:22 -0500
Subject: [PATCH] ENGR00117344 fix mono record during stereo playback

Starting a mono record stream and doing stereo playback resulted in
wrong sample rate.  Also starting stereo playback then doing mono
record.

Signed-off-by: Alan Tull <r80115@freescale.com>
---
 sound/soc/codecs/sgtl5000.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/codecs/sgtl5000.c b/sound/soc/codecs/sgtl5000.c
index 9910887..a63b65c 100644
--- a/sound/soc/codecs/sgtl5000.c
+++ b/sound/soc/codecs/sgtl5000.c
@@ -886,7 +886,7 @@ struct snd_soc_dai sgtl5000_dai = {
 		     },
 	.capture = {
 		    .stream_name = "Capture",
-		    .channels_min = 1,
+		    .channels_min = 2,
 		    .channels_max = 2,
 		    .rates = SGTL5000_RATES,
 		    .formats = SGTL5000_FORMATS,
-- 
1.5.4.4

